aws-java-sdk-core/src/main/java/com/amazonaws/ClientConfiguration.java

```diff
@@ -616,6 +618,48 @@ private String getSystemProperty(String property) {
         return System.getProperty(property);
     }
 
+    /**
+     * Returns the value for the given environment variable.
+     */
+    private String getEnvironmentVariable(String variable) {
+        return System.getenv(variable);
+    }
+
+    /**
+     * Returns a URL object for the given environment variable, if can be constructed, null otherwise.
+     */
+    private URL getUrlFromEnvironmentVariable(String variable) {
```

Comment:
Can we rename variable to environmentVariable? Here and in all new places.

Reply:
Good catch, fixed in the newest commit